### PR TITLE
[Admin] Changed product code to variant code on orders show page

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | yes
| BC breaks?      | no
| Related tickets | fixes #6116
| License         | MIT

### CODE REVIEW

File: features/order/managing_orders/seeing_order_item_detailed_data.feature [hunk 1]
```diff
@@ -25,7 +25,8 @@ Feature: Seeing order item detailed data
     Scenario: Seeing details of item in one row
         Given I view the summary of the order "#00000666"
         When I check "Iron Man T-Shirt" data
-        Then its unit price should be $39.00
+        Then its code should be "IRON_MAN_T-SHIRT"
```
Comment (pamil) [inline features/order/managing_orders/seeing_order_item_detailed_data.feature]: Where is it defined?

Comment (michalmarcinkowski) [inline features/order/managing_orders/seeing_order_item_detailed_data.feature]: To verify this fix is correct we should add two variants of the same product and validate their codes.

File: features/order/managing_orders/seeing_order_item_detailed_data.feature [hunk 2]
```diff
@@ -8,27 +8,30 @@ Feature: Seeing order item detailed data
         Given the store operates on a single channel in "United States"
         And the store has "VAT" tax rate of 10% for "T-Shirts" within the "US" zone
         And the store classifies its products as "T-Shirts"
-        And the store has a product "Iron Man T-Shirt" priced at "$39.00"
-        And it belongs to "T-Shirts" tax category
+        And the store has a product "Marvel T-Shirt"
+        And this product has "Iron Man T-Shirt" variant priced at "$49.00"
+        And this product has "Thor T-Shirt" variant priced at "$39.00"
```
Comment (TheMadeleine) [inline features/order/managing_orders/seeing_order_item_detailed_data.feature]: ...**the** "XXX" variant...

Comment (NeverResponse) [inline features/order/managing_orders/seeing_order_item_detailed_data.feature]: Not in scope of this PR.
Besides most of our steps need a hardcore semantical rework

**Comment by michalmarcinkowski** [comment]
Thanks Janek! 👍

**Pull request merged**